models/intel/smartlab-object-detection-0004/accuracy-check.yml

```diff
@@ -0,0 +1,37 @@
+models:
+  - name: smartlab-object-detection-0001
+
+    launchers:
+      - framework: dlsdk
+        adapter: 
+          type: yolo_v3
+          classes: 3
+          anchors: yolo_v3
+
+    datasets:
+      - name: DSI1867
```

Comment:
it is internal dataset name at intel, i do not think that it should provided to public configuration file